### PR TITLE
setNextMonth() and setPrevMonth() fixes and other additions

### PR DESCRIPTION
**Pull request by brianlizhou**
Added a few parameters and methods to constructors and objects.
Test the calendar in my personal Sakai via tomcat and no new bugs have been created from 
what I have tested

### CODE REVIEW

**Comment by ncaidin** [comment]
ICLA on file as Brian Zhou .  
http://licensing.apereo.org/

**Comment by ottenhoff** [comment]
Hi Brian,

Thanks for submitting a PR to the Sakai project! Please checkout our contribution guidelines here:

  https://github.com/sakaiproject/sakai/blob/master/CONTRIBUTING.md

A few notes:

 * You should reference a JIRA in jira.sakaiproject.org
 * Your pull request should reference the JIRA
 * Your commit message should reference the JIRA
 * Your pull request should contain one commit only. Please squash and rebase your commits.

**Comment by buckett** [comment]
You're Jira https://jira.sakaiproject.org/browse/SAK-30400 seems to be about the Calendar tool (from the screenshot), but your changes here are in the Calendar summary tool. Could you explain more in the Jira what these code changes are fixing.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+FROM library/ubuntu:16.04
```
Comment (steveswinsburg) [inline Dockerfile]: You should remove this from the PR

File: Test [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Docker Test
```
Comment (steveswinsburg) [inline Test]: And remove this from the PR also

**Comment by buckett** [comment]
Closed as changes weren't cleaned up, if you tidy this up please feel free to re-open it.

**Pull request closed without merging**